context/app/static/js/shared-styles/inputs/MultiAutocomplete/MultiAutocomplete.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+
+import Autocomplete, { AutocompleteProps } from '@mui/material/Autocomplete';
+import TextField, { TextFieldProps } from '@mui/material/TextField';
+import { InputLabelProps as inputLabelProps } from '@mui/material/InputLabel';
+
+interface MultiAutocompleteTypes extends AutocompleteProps {
+  tagComponent: React.ReactNode;
+  renderInputProps: TextFieldProps;
```

Comment:
`TextFieldProps` is also generic based on which variant prop is used; since we're using `outlined`, we should be able to set this to `TextFieldProps<'outlined'>`.
```suggestion
  renderInputProps: TextFieldProps<'outlined'>;
```